http/headers/Content-Security-Policy.json

```diff
@@ -1036,6 +1036,42 @@
               }
             }
           },
+          "trusted-types-eval": {
+            "__compat": {
+              "description": "`trusted-types-eval` source expression",
+              "spec_url": "https://w3c.github.io/webappsec-csp/#grammardef-trusted-types-eval",
+              "tags": [
+                "web-features:csp",
+                "web-features:trusted-types"
+              ],
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
```

Comment:
This isn't yet implemented in the Firefox implementation hence I've not copied the flag data over.